tsconfig.json

```diff
@@ -10,6 +10,13 @@
     "importHelpers": true,
     "jsx": "react",
     "resolveJsonModule": true,
+    "baseUrl": ".",
+    "paths": {
+      "vega-lite": ["../../vega-lite/src"],
+      "vega-lite/*": ["../../vega-lite/src/*"],
```

Comment:
does that actually work since we only allow some exports defined in https://github.com/vega/vega-lite/blob/473f3594e3db5c24f439f0c183eac8f662f46817/package.json#L27 ?